<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="startbtn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>530</y>
      <width>80</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>開始遊戲</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitbtn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>530</y>
      <width>80</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>離開遊戲</string>
    </property>
   </widget>
   <widget class="QLabel" name="counter">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>時間</string>
    </property>
   </widget>
   <widget class="QLabel" name="score">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>230</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>分數</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="counterLCD">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="scoreLCD">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>230</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="waitTXT">
    <property name="geometry">
     <rect>
      <x>684</x>
      <y>508</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="finalscoreTXT">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>170</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>最終分數</string>
    </property>
   </widget>
   <widget class="QLabel" name="finalscore">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>220</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="againbtn">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>530</y>
      <width>80</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>再次遊戲</string>
    </property>
   </widget>
   <widget class="QLabel" name="loading">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>530</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>遊戲載入中...</string>
    </property>
   </widget>
   <widget class="QLabel" name="highscoreTXT">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>最高得分</string>
    </property>
   </widget>
   <widget class="QLabel" name="highscore">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="good">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>151</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/good.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="ok">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>161</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/ok.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="miss">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/miss.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="goodicon">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>300</y>
      <width>81</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/goodicon.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="okicon">
    <property name="geometry">
     <rect>
      <x>354</x>
      <y>310</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/okicon.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="missicon">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>310</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/missicon.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="combobar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/combobar.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="combo1num0">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>280</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/num0.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="combo1num1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/num1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="combo1num2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/num2.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="combo1num3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>330</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/num3.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="combo1num4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>350</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/num4.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="combo1num5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>370</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/num5.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="combo1num6">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>388</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/num6.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="combo1num7">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>399</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/num7.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="combo1num8">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>400</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/num8.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="combo1num9">
    <property name="geometry">
     <rect>
      <x>364</x>
      <y>410</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/num9.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="combo10num0">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/num0.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="combo10num1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>350</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/num1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="combo10num2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>370</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/num2.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="combo10num3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>390</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/num3.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="combo10num4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>420</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/num4.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="combo10num5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>440</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/num5.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="combo10num6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>460</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/num6.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="combo10num7">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>480</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/num7.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="combo10num8">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>500</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/num8.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="combo10num9">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>520</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/num9.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="result">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>280</y>
      <width>101</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/result.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="resultsuccess">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>290</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="resultfail">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>360</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="goodTXT">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>250</y>
      <width>41</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/goodicon.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="okTXT">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>320</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/img/okicon.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="resultgood">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>270</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="resultok">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>340</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="comboTXT">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>160</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>最大連段</string>
    </property>
   </widget>
   <widget class="QLabel" name="combo">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>200</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
